locksmith/src/operations/lockOperations.ts

```diff
@@ -106,6 +113,34 @@ export const getLockIcon = async ({
   }
 }
 
+export const getKeyIcon = async ({
+  network,
+  lockAddress,
+  keyId,
+}: {
+  network: number
+  lockAddress: string
+  keyId: string
+}): Promise<IconType | undefined> => {
+  console.log({ lockAddress, keyId })
+
+  // Temporary icon for FilBangalore
+  if (
+    network == 42161 &&
+    lockAddress.toLowerCase() === '0x02c510be69fe87e052e065d8a40b437d55907b48'
+  ) {
+    const icon = await ticketForFilBangalore({
+      network,
+      lockAddress,
+      tokenId: keyId,
+    })
+
+    return { type: 'image/svg+xml', icon, isURL: false }
+  }
+  // By default, no icon
+  return
```

Comment:
Nitpick but I think returning `null` makes more sense since it means the function always returns a object instead of sometimes it does and sometimes it doesn't at all.

Reply:
fair point!